keras_core/saving/legacy/saved_model/json_utils.py

```diff
@@ -0,0 +1,219 @@
+"""JSON utilities for legacy saving formats (h5 and SavedModel)"""
+
+import collections
+import enum
+import functools
+import json
+
+import numpy as np
+
+from keras_core.saving import serialization_lib
+from keras_core.saving.legacy import serialization
+from keras_core.utils.module_utils import tensorflow as tf
+
+if tf.available:
```

Comment:
Don't do this at the top of the file, since it would cause a TF import when importing `keras_core` regardless of whether the user is using it. In-line it where it gets used.

Reply:
Fixed, in-lined the import.